doc/03-omnifest/02-external.md

```diff
@@ -4,20 +4,16 @@
 
 These directives are only allowed within a [`otk.target.osbuild.<name>`](./01-directive.md#otktargetconsumername).
 
-### `otk.external.osbuild.depsolve-dnf4`
+### `otk.external.osbuild.depsolve-dnf{4,5}`
 
 Solves a list of package specifications to RPMs and specifies them in the
 osbuild manifest as sources.
 
-### `otk.external.osbuild.depsolve-dnf5`
-
-Expects a `map` as its value.
+### `otk.external.osbuild.file-from-text`
```

Comment:
(nitpick) neither this nor the one below are actually used yet, I think would be ideal if all our directives have a use-case